engine/graphics/src/graphics.cpp

```diff
@@ -271,6 +276,11 @@ namespace dmGraphics
         return g_functions.m_NewContext(params);
     }
 
+    HContext GetInstalledContext()
+    {
+        assert(g_adapter && "No graphics adapter installed");
+        return g_adapter->m_GetContextCb();
+    }
```

Comment:
This is currently only available internally (via graphics_private.h) to be able to get the current window handle. I decided to put a callback on the adapter pointer to avoid having to add a new function signature on the adapter function table, but with the benefit that each adapter will get compile errors if no callback is passed in.